### ISSUE TITLE
Regular Expression Denial of Service in debug

### ISSUE THREAD

**Issue by VMBindraban** [microsoft/appcenter-sdk-react-native#1019]
### **Description**

Regular Expression Denial of Service in `debug` - https://github.com/advisories/GHSA-gxpj-cx7g-858c

Seems https://github.com/microsoft/appcenter-sdk-react-native/issues/743 is back.

### **Details**

1. Which version of the App Center SDK are you using?
    - `5.0.0`

```
node_modules/appcenter-link-scripts/node_modules/debug
  appcenter-link-scripts  >=2.5.0
  Depends on vulnerable versions of debug
  node_modules/appcenter-link-scripts
    appcenter  >=2.5.0
    Depends on vulnerable versions of appcenter-link-scripts
    node_modules/appcenter
      appcenter-analytics  >=2.5.0
      Depends on vulnerable versions of appcenter
      node_modules/appcenter-analytics
      appcenter-crashes  >=2.5.0
      Depends on vulnerable versions of appcenter
      node_modules/appcenter-crashes

**Comment by DmitriyKirakosyan** [comment]
@VMBindraban , thank you for reaching out!
We've already fixed and merged it here https://github.com/microsoft/appcenter-sdk-react-native/pull/1018.
However, it is not released yet. We will update this thread once the fix is released.

**Comment by nathfy** [comment]
@DmitriyKirakosyan Any idea when the next release is due? as both this and inflight and flagged by snyk:
![image](https://github.com/microsoft/appcenter-sdk-react-native/assets/1806224/3a186193-4c4f-426f-8823-5c9db453c514)

**Comment by AnatolyPristensky** [comment]
Hello @nathfy this fix available in new [5.0.1](https://github.com/microsoft/appcenter-sdk-react-native/releases/tag/5.0.1) version.
